code/modules/clothing/suits/hood.dm

```diff
@@ -2,38 +2,21 @@
 
 /obj/item/clothing/suit/hooded
 	actions_types = list(/datum/action/item_action/toggle)
-	var/obj/item/clothing/head/hood
-	var/hoodtype = /obj/item/clothing/head/winterhood //so the chaplain hoodie or other hoodies can override this
-
-/obj/item/clothing/head/chaplain_hood
-	flags = NODROP
-
-/obj/item/clothing/head/nun_hood
-	flags = NODROP
-
-/obj/item/clothing/head/monk_hood
-	flags = NODROP
-
-/obj/item/clothing/head/culthood
-	flags = NODROP
-
-/obj/item/clothing/head/culthood/alt
-	flags = NODROP
-
-/obj/item/clothing/head/berserkerhood
-	flags = NODROP
+	var/obj/item/clothing/head/hooded/hood
+	var/hoodtype = /obj/item/clothing/head/hooded/winterhood //so the chaplain hoodie or other hoodies can override this
 
 /obj/item/clothing/suit/hooded/New()
+	. = ..()
 	MakeHood()
-	..()
 
 /obj/item/clothing/suit/hooded/Destroy()
+	. = ..()
```

Comment:
I'm skeptical of doing this, because the `hood` will be in the suit's `contents`, which means it'll get `qdel`'d then. Makes more sense to delete the hood first.